server/schemas/item.graphql

```diff
@@ -47,7 +44,7 @@ type ItemPayload {
 }
 
 type DeleteItemPayload {
-  itemId: ID!
+  itemID: ID!
```

Comment:
```suggestion
  itemId: ID!
```

In GraphQL, `hogehogeID` shoud be `hogehogeId`
In Golang, `hogehogeId` should be `hogehogeID`